src/Components/WebAssembly/Authentication.Msal/src/Interop/AuthenticationService.ts

```diff
@@ -326,6 +364,7 @@ export class AuthenticationService {
         if (!AuthenticationService._initialized) {
             AuthenticationService._initialized = true;
             AuthenticationService.instance = new MsalAuthorizeService(settings);
+            await AuthenticationService.instance.initializeMsalHandler();
```

Comment:
Can you check what we do on the C# side of things? I think there might be a small bug here WRT to async. I believe on the oidc-client.js implementation we hold on to the promise.

Reply:
Ah yeah, makes sense. Updated this to return the promise.